### ISSUE TITLE
Llama-3

### ISSUE THREAD

**Issue by wj210** [sail-sg/I-FSJ#2]
Hi, do i run exps.rs_llama3 for newer models like llama-3.1 and deepseek?

Also, how did u get the loss for GPT4? does their api provide this service?

Thanks

**Comment by xszheng2020** [comment]
Hi, @wj210 
1. do i run exps.rs_llama3 for newer models like llama-3.1 and deepseek?
Yes, you can test the code on newer llama models but please check whether the template is consistent with their official template

2. how did u get the loss for GPT4? does their api provide this service?
Yes, gpt-4 api can provide the logits of the top-20 tokens
